resources/linux/org.lite_xl.lite_xl.appdata.xml

```diff
@@ -29,6 +29,6 @@
   </provides>
 
   <releases>
-    <release version="2.0.1" date="2021-08-28" />
+    <release version="2.0.1" date="2021-11-01" />
```

Comment:
I wonder if it would be better to add a new `<release ...>` instead of replacing it.

Reply:
Also, version and date seem a bit... off.